src/Workspaces/Core/Portable/PatternMatching/PatternMatcher.cs

```diff
@@ -174,106 +175,96 @@ private static bool ContainsUpperCaseLetter(string pattern)
-                if (caseInsensitiveIndex > 0)
-                {
-                    // We found the pattern somewhere in the candidate.  This could be a substring match.
-                    // However, we don't want to be overaggressive in returning just any substring results.
-                    // So do a few more checks to make sure this is a good result.
+                // We found the pattern somewhere in the candidate.  This could be a substring match.
+                // However, we don't want to be overaggressive in returning just any substring results.
+                // So do a few more checks to make sure this is a good result.
 
-                    if (!patternIsLowercase)
-                    {
-                        // Pattern contained uppercase letters.  This is a strong indication from the
-                        // user that they expect the same letters to be uppercase in the result.  As 
-                        // such, only return this if we can find this pattern exactly in the candidate.
+                if (!patternIsLowercase)
+                {
+                    // Pattern contained uppercase letters.  This is a strong indication from the
+                    // user that they expect the same letters to be uppercase in the result.  As 
+                    // such, only return this if we can find this pattern exactly in the candidate.
 
-                        var caseSensitiveIndex = _compareInfo.IndexOf(candidate, patternChunk.Text, CompareOptions.None);
-                        if (caseSensitiveIndex > 0)
-                        {
-                            if (char.IsUpper(candidate[caseInsensitiveIndex]))
-                            {
-                                return new PatternMatch(
-                                    PatternMatchKind.StartOfWordSubstring, punctuationStripped, isCaseSensitive: true,
-                                    matchedSpan: GetMatchedSpan(caseInsensitiveIndex, patternChunk.Text.Length));
-                            }
-                            else
-                            {
-                                return new PatternMatch(
-                                    PatternMatchKind.NonLowercaseSubstring, punctuationStripped, isCaseSensitive: true,
-                                    matchedSpan: GetMatchedSpan(caseSensitiveIndex, patternChunk.Text.Length));
-                            }
-                        }
-                    }
-                    else
+                    var caseSensitiveIndex = _compareInfo.IndexOf(candidate, patternChunk.Text, CompareOptions.None);
+                    if (caseSensitiveIndex > 0)
                     {
-                        // Pattern was all lowercase.  This can lead to lots of hits.  For example, "bin" in
-                        // "CombineUnits".  Instead, we want it to match "Operator[|Bin|]ary" first rather than
-                        // Com[|bin|]eUnits
```

Comment:
❔ It looks like this comment was removed. Was this intentional, or perhaps an artifact of the way GitHub rendered the diff?

Reply:
yeah.  i must have accidentally done that.  restoring.